FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Posix/Makefile

```diff
@@ -52,34 +53,42 @@ SOURCE_FILES += ${FREERTOS_DIR}/Source/portable/MemMang/heap_3.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_Sockets.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_Stream_Buffer.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_TCP_IP.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_TCP_Reception.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_TCP_State_Handling.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_TCP_Transmission.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_TCP_Utils.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_TCP_WIN.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_Tiny_TCP.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/FreeRTOS_UDP_IP.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/portable/BufferManagement/BufferAllocation_2.c
-SOURCE_FILES += ${FREERTOS_PLUS_DIR}/Source/FreeRTOS-Plus-TCP/source/portable/NetworkInterface/linux/NetworkInterface.c
-
-
-CFLAGS 			:= -ggdb3
-LDFLAGS			:= -ggdb3 -pthread -lpcap
-CPPFLAGS		:=    $(INCLUDE_DIRS) -DBUILD_DIR=\"$(BUILD_DIR_ABS)\"
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_ARP.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_DHCP.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_DNS.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_DNS_Cache.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_DNS_Callback.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_DNS_Networking.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_DNS_Parser.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_ICMP.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_IP.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_IP_Timers.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_IP_Utils.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_Sockets.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_Stream_Buffer.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_TCP_IP.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_TCP_Reception.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_TCP_State_Handling.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_TCP_Transmission.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_TCP_Utils.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_TCP_WIN.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_Tiny_TCP.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/FreeRTOS_UDP_IP.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/portable/BufferManagement/BufferAllocation_2.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/portable/NetworkInterface/libslirp/MBuffNetifBackendLibslirp.c
+SOURCE_FILES += ${FREERTOS_PLUS_TCP_DIR}/portable/NetworkInterface/libslirp/MBuffNetworkInterface.c
+
+CFLAGS 			:= -ggdb3 
+LDFLAGS			:= -ggdb3 -pthread
+
+CFLAGS += $(shell pkg-config --cflags slirp)
```

Comment:
Add a comment above here explaining these commands add in the header include paths, and the library flags